### ISSUE TITLE
lifting root of repo design constraint

### ISSUE THREAD

**Issue by softprops** [leandroBorgesFerreira/dag-command#28]
from this project's readme

> This module assumes that the modules are in the folders in the root of the project

I'd like to use your plugin but within my git repo my gradle build exists under `{repo-root}/modules/` which I believe is breaking the assumptions of this plugin. would it be possible to add a configuration property to make this assumption a flexible one? something like 

```kts
  dagCommand {
    filter = "all"
    defaultBranch = "origin/main"
    outputType = "json"
    printModulesInfo = true
    projectPrefix = "modules/" // <- strips this off in git diff calculation
  }
```

**Comment by leandroBorgesFerreira** [comment]
Hello @softprops. 
Sorry for the late reply, I didn't see your issue =|. 

Your request makes perfect sense. More than this... the limitation of where the modules should be should be improved. I need to find a way to support modules in any place of the project. 

I'll do some research and update this issue.

**Comment by leandroBorgesFerreira** [comment]
It has been a while, but this is fixed in the version 1.10.0. Now you can choose any module that you prefer.
